examples/java/core/runtime_schema/src/main/java/org/bondlib/examples/RuntimeSchemaExample.java

```diff
@@ -0,0 +1,44 @@
+package org.bondlib.examples;
+
+import java.io.ByteArrayInputStream;
+import java.io.ByteArrayOutputStream;
+import java.io.IOException;
+
+import org.bondlib.*;
+
+import org.bondlib.protocol.CompactBinaryReader;
+import org.bondlib.protocol.CompactBinaryWriter;
+
+// See build.gradle for namespace mapping
+import org.bondlib.examples.runtime_schema.Example;
+
+public class RuntimeSchemaExample {
+
+    public static void main(final String[] args) throws IOException {
+
+        final ByteArrayOutputStream output = new ByteArrayOutputStream();
+        final CompactBinaryWriter writer = new CompactBinaryWriter(output, 1);
+
+        final Serializer<SchemaDef> serializer = new Serializer<>();
+        serializer.serialize(Example.BOND_TYPE.buildSchemaDef(), writer);
+
+        final ByteArrayInputStream input = new ByteArrayInputStream(output.toByteArray());
+
+        final CompactBinaryReader reader = new CompactBinaryReader(input, 1);
+        final Deserializer<SchemaDef> deserializer = new Deserializer<>(SchemaDef.BOND_TYPE);
+
+        final SchemaDef schemaDef = deserializer.deserialize(reader);
+        final RuntimeSchema schema = new RuntimeSchema(schemaDef);
```

Comment:
I thought we're getting rid of RuntimeSchema. It seems to be unused, except in examples.

Reply:
Yeah, I missed the examples. Correction incoming.